pkgs/development/tools/pyre/default.nix

```diff
@@ -3,12 +3,12 @@
 let
   # Manually set version - the setup script requires
   # hg and git + keeping the .git directory around.
-  pyre-version = "0.0.17";  # also change typeshed revision below with $pyre-src/.typeshed-version
+  pyre-version = "0.0.18";  # also change typeshed revision below with $pyre-src/.typeshed-version
```

Comment:
Did you read the comment?

Reply:
It is still the same: https://github.com/facebook/pyre-check/blob/v0.0.18/.typeshed-version

Reply:
Thanks!